miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/workflow/DefaultWorkflowManager.java

```diff
@@ -60,7 +60,10 @@ public Workflow beginWorkflow(String workflowNameString) throws IOException {
 
   @Override
   public Workflow processInput(Workflow workflow, int stepNumber, String input) throws IOException {
-    workflow.processInput(stepNumber, makeProgressStep(input, workflow.getStep(stepNumber).getInputTypes()));
+    List<String> errors = workflow.processInput(stepNumber, makeProgressStep(input, workflow.getStep(stepNumber).getInputTypes()));
+    if (!errors.isEmpty()) {
+      throw new ValidationException(errors.stream().map(err -> new ValidationError(err)).collect(Collectors.toList()));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)